File: example-codes-py/blackjack-and-a-deck-of-cards/blackjack_card.py
from .card import *


class BlackjackCard(Card):
    def __init__(self, suit, face_value):
        super().__init__(suit, face_value)
        self.__game_value = face_value
        if self.__game_value > 10:
            self.__game_value = 10

    def get_game_value(self):
        return self.__game_value

File: example-codes/blackjack-and-a-deck-of-cards/java/BlackjackCard.java
package blackjack;

public class BlackjackCard extends Card {
    private int gameValue;

    public BlackjackCard(SUIT suit, int faceValue) {
        super(suit, faceValue);
        this.gameValue = faceValue;
        if (this.gameValue > 10) {
            this.gameValue = 10;
        }
    }

    public int getGameValue() {
        return gameValue;
    }
}

